sdk/storagecache/Azure.ResourceManager.StorageCache/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why do we need this file and another `package-lock.json` file in StorageCache?

Reply:
hmm... it's checked in by mistake

Reply:
fixed